### PR TITLE
remove unnecessary `.call` and cleanup

### PR DESCRIPTION
**Pull request by bekzod**

### CODE REVIEW

File: packages/ember-metal/lib/property_events.js [hunk 1]
```diff
@@ -248,7 +248,7 @@ function changeProperties(callback, binding) {
   try {
     callback.call(binding);
   } finally {
-    endPropertyChanges.call(binding);
```
Comment (bekzod) [inline packages/ember-metal/lib/property_events.js]: `endPropertyChanges` does not need `.call`

**Review by krisselden — APPROVED** [review]

**Comment by krisselden** [comment]
Has failing tests

**Comment by bekzod** [comment]
thanks checking

**Comment by locks** [comment]
@bekzod can you check the conflicts please? :)

**Comment by bekzod** [comment]
@locks ofc 👍

**Comment by bekzod** [comment]
needs restart

**Comment by stefanpenner** [comment]
restarting sauce.

**Comment by krisselden** [comment]
looks good.

**Pull request merged**